package.json

```diff
@@ -1,29 +1,8 @@
 {
   "name": "remark-preset-prettier",
   "version": "1.0.2",
-  "type": "module",
-  "description": "Turns off all rules that are unnecessary or might conflict with Prettier.",
-  "repository": "git+https://github.com/remarkjs/remark-preset-prettier.git",
-  "author": "JounQin <admin@1stg.me>",
-  "funding": {
-    "type": "opencollective",
-    "url": "https://opencollective.com/unified"
-  },
+  "description": "remark preset to turn rules off that conflict with prettier",
   "license": "MIT",
-  "packageManager": "yarn@1.22.19",
-  "engines": {
-    "node": ">=14.8"
```

Comment:
The engine is required for top level await, isn't it?

Reply:
The `engines` field is a *hint* for package managers.
In our experience, they have been incorrect often.
For example, Node 10 with a flag, and Node 12+ also support ESM.
Node also supports top-level await with a flag earlier.
Other engines are also supported: bun, deno.

When engines are wrong, it makes it very hard to use a package.
I strongly recommend not using them. If the code doesn’t work in an engine, then users will figure that out. No need for npm/yarn/pnpm to force that.

Reply:
Then we should set it correctly. It's not a good solution to ignore IMO. A flag is experimental, the users should always use `ignore-engines` in this case.

For `deno`, `bun`, the `node` engine doesn't affect. And We're focusing on Node by default, aren't we?

Reply:
For more info on why we prefer not using `engines`, see previous discussions:
- https://github.com/unifiedjs/unified/issues/70
- https://github.com/mdx-js/mdx/issues/878

> Then we should set it correctly.

I don’t believe that it is possible to get engines right, due to:
- dependencies
- alpha/beta/rcs of engines
- flags of engines

Given that:
- it can’t be right always,
- the packages maintained by us are typically deep down in people’s dependency trees,
- the results of incorrect engines is very annoying (especially for deep down packages that can’t be “fixed” even though they work)

…we prefer not adding `engines`. Instead, we describe compatibility in the readme. For example: https://github.com/remarkjs/remark-gfm#compatibility.

> And We're focusing on Node by default, aren't we?

I guess this project doesn’t work anywhere other than Node. So, sure.
However, most all of our packages work everywhere.

> A flag is experimental, the users should always use ignore-engines in this case.

Why?

Reply:
OK, I give up if this has been discussed a lot of times.

> Why?

Because they are using experimental features. 🤣 In my projects, engines are always for stable features.